services/github/github-package-registry-downloads.tester.js

```diff
@@ -0,0 +1,148 @@
+'use strict'
+
```

Comment:
There's 18 tests in here, some of which folks may find duplicative. I don't feel too terribly strongly, but here was my thinking behind including them:

* There's three different GraphQL queries that are issued depending on whether the route includes a `version`, and whether the `version` value is `latest`
* The schema has two different contracts for package statistics, that currently have the same field names: [RegistryPackageStatistics](https://developer.github.com/v4/object/registrypackagestatistics/) and [RegistryPackageVersionStatistics](https://developer.github.com/v4/object/registrypackageversionstatistics/). Obviously those can change/evolve independently so I think it's important that our tests cover each interval variant for each of those contracts
* GitHub Package Registry is still in a beta (seemingly a private one) which AFAICT means that the contracts are still subject to change, and the GitHub V4 GraphQL API that's used by this service is still [undergoing contract changes](https://developer.github.com/v4/breaking_changes/) as well. This is one of those instances where I'd prefer to have a few extra tests so that we're more likely to see breaking API changes more quickly.